autoload/go/lint.vim

```diff
@@ -126,9 +126,13 @@ function! go#lint#Vet(bang, ...) abort
   endif
 
   if a:0 == 0
-    let [l:out, l:err] = go#util#Exec(['go', 'vet', go#package#ImportPath()])
+    let [l:out, l:err] = go#util#Exec(['go', 'vet', '-tags', go#config#BuildTags(), go#package#ImportPath()])
   else
-    let [l:out, l:err] = go#util#Exec(['go', 'vet'] + a:000 + [go#package#ImportPath()])
+    if empty(go#config#BuildTags())
+      let [l:out, l:err] = go#util#Exec(['go', 'vet'] + a:000 + [go#package#ImportPath()])
+    else
+      let [l:out, l:err] = go#util#Exec(['go', 'vet', '-tags', go#config#BuildTags()] + a:000 + [go#package#ImportPath()])
+    endif
```

Comment:
Can you simplify this if/else to be more similar to the pattern in https://github.com/fatih/vim-go/blob/master/autoload/go/def.vim#L31-L34 ?

Reply:
Will, do! Thanks for the link 👍